plugins/rapid7_insightidr/unit_test/mock.py

```diff
@@ -91,8 +98,8 @@ def mock_request_put(url: str) -> MockResponse:
     if url == f"{Util.STUB_URL_API}/idr/v2/investigations/{STUB_INVESTIGATION_IDENTIFIER}/status/{STUB_STATUS}":
         return MockResponse("update_investigation", 200)
     if (
-        url
-        == f"{Util.STUB_URL_API}/idr/v2/investigations/{STUB_INVESTIGATION_IDENTIFIER}/disposition/{STUB_DISPOSITION}"
+            url
+            == f"{Util.STUB_URL_API}/idr/v2/investigations/{STUB_INVESTIGATION_IDENTIFIER}/disposition/{STUB_DISPOSITION}"
```

Comment:
Was this black formattings fault or has it just be indented a lot? If possible bring it back but if it's due to `black` then leave it as is